### DISCUSSION TITLE
Reset Multiplier on Count Value

### DISCUSSION THREAD

**Discussion by boilermanc** [arendst/Tasmota#15323] [Q&A] [answered]
### PROBLEM DESCRIPTION
I am setting up a wemos d1 mini with the latest version of the sensor release (just loaded it tonight). I am using it with a water flow sensor. I need to reset the multiplier in the count value. I have read through a few post but not really finding what I am after. I found a great write up from Michael McSharry where he set the gpio 14 to counter 1. He then had more options to manage the counter. I have tried the same but I do not see those as options. Is there a different version of tasmota that would have those options in there? I have my wemos set as Generic (18).

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ ] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._
This was from a document from Michael. 

<img width="267" alt="Counter1" src="https://user-images.githubusercontent.com/656498/162552407-26ad2ef5-3550-447e-bdf5-b5f5ca88c1f6.PNG">

### ADDITIONAL CONTEXT
_Add any other context about the problem here._
If there is another way to set this, please let me know.

thanks!

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by ascillato** [comment, marked as answer]
Hi,

That screenshot is nor standard neither oficial Tasmota. Seems to be a custom made version. Did you have the link of the site you are referring to?

Those options are not available in oficial Tasmota. But using standard Tasmota you can add a rule to reset the counter as needed. I also use water sensors with counters and my rule is:

```JS
Teleperiod 60
Rule1 1
Rule1 ON Tele-Counter#c1 DO Counter 0 ENDON
```

With that, I got the flow by minute to be displayed in my Dashboard.

**Reply by sfromis** [reply]
When it comes to resetting counters, there is also SetOption79 to "Reset counters at TelePeriod time".

Not sure that that multiplier (which is not part of Tasmota) does, but it may be feasible to do something similar with rules, or in your backend.

**Reply by boilermanc** [reply]
Thanks very much for the quick reply. That was kinda what I was thinking. I can try to hunt down the owner just to get an understanding of how they did that. In the meantime, i will try working with the rules. I have not done that with Tasmota yet so will learn something new. Might be back with more questions!

Very helpful! 

thanks!

**Reply by boilermanc** [reply]
and here was the link to the doc. very impressive doc!

[http://mcssprinklers.com/mcsMQTT.pdf](url)

**Reply by sfromis** [reply]
Related: https://github.com/mcsSolutions

**Reply by barbudor** [reply]
Not mush there : "mcsSolutions doesn't have any public repositories yet"

**Reply by sfromis** [reply]
Right, but if someone wants to ping

**Reply by barbudor** [reply]
Page 305 
> The firmware, which is the same Tasmota-variant firmware described for devices in this manual is at
http://mcsSprinklers.com/mcsTasmota.zip. This same firmware can be used for all other applications of
the Sonoff devices in this manual as of June 2018.
The source, based upon Tasmota 5.9.1 can also be made available for anyone interested. There is also a
firmware based upon Tasmota 6.0.0a that is available under mcsTasmota6
http://mcsSprinklers.com/mcsTasmota6.zip. This contains only the addition of the irrigation control
logic.

This is legally not against the GPL Terms (Article 6.c) but IMHO not very friendly to not publicaly republish the modified source

**Reply by sfromis** [reply]
Ideally having it as a Github fork
